smart_card_connector_app/src/window-apps-displaying.js

```diff
@@ -43,11 +43,14 @@ const KnownApp = PermissionsChecking.KnownApp;
 /** @type {!goog.log.Logger} */
 const logger = GSC.Logging.getScopedLogger('ConnectorApp.MainWindow');
 
-/** @type {!Element} */
-const appListElement =
-    /** @type {!Element} */ (goog.dom.getElement('app-list'));
-
-const knownAppsRegistry = new PermissionsChecking.KnownAppsRegistry();
+//#revisitcode
+/**
+ * @type {!Element}
+ * @const
+ */
+// var appListElement = /** @type {!Element} */ (goog.dom.getElement('app-list'));
+//revisitcode
+// const knownAppsRegistry = new PermissionsChecking.KnownAppsRegistry();
```

Comment:
Hmm, do you remember why this had to be commented out? That's strange if that class doesn't work in extensions, but I might be missing something.